features/upgrade/sdn/externalIP-upgrade.feature

```diff
@@ -60,12 +60,18 @@ Feature: SDN externalIP compoment upgrade testing
   @proxy @noproxy @disconnected @connected
   Scenario: Check the externalIP works well after upgrade
     Given I switch to cluster admin pseudo user
-    Given I store the schedulable nodes in the :nodes clipboard
-    And the Internal IP of node "<%= cb.nodes[0].name %>" is stored in the :hostip clipboard
+    # Get the external ip from  service
+    When I run the :get client command with:
+      | resource      | service                         |
+      | resource_name | service-unsecure                |
+      | n             | externalip-upgrade              |
+      | o             | jsonpath={.spec.externalIPs[0]} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :hostip clipboard
```

Comment:
Can be shorten as
```
When I use the "externalip-upgrade" project
And evaluation of `service('xxx')spec.externalIPs[0]` is stored in the :yyy clipboard
```

Reply:
@liangxia Thanks for the suggestion.  Seems not working.
'''
    And evaluation of `service('service-unsecure').spec.externalIPs[0]` is stored in the :hostip clipboard # features/step_definitions/common.rb:128
      undefined method `spec' for #<BushSlicer::Service:0x00007f8021063620>
'''

Reply:
Updated it.